docs/src/components/ExampleSnippet/renderElementToJSX.ts

```diff
@@ -14,7 +14,7 @@ const renderElementToJSX = (element: React.ReactNode, triggerErrorOnRenderFn: bo
       (typeof element.type === 'function' ? 'NoDisplayName' : element.type),
     showDefaultProps: false,
     showFunctions: true,
-    filterProps: ['accessibility'],
+    filterProps: ['accessibility', 'onClick', 'onChange'],
```

Comment:
👍  maybe in the future we can make this filtering smarter, but definitely now we need them

Reply:
Yep, I thinking about it and how we can avoid this filtering at all